<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="1" ignored="0" total="8" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="117940" started-at="2021-03-08T08:26:01 IST" finished-at="2021-03-08T08:27:59 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="117940" started-at="2021-03-08T08:26:01 IST" finished-at="2021-03-08T08:27:59 IST">
      <class name="testScript.CallLogLookerReport">
        <test-method status="PASS" signature="test1()[pri:1, instance:testScript.CallLogLookerReport@3701eaf6]" name="test1" duration-ms="70854" started-at="2021-03-08T08:26:01 IST" finished-at="2021-03-08T08:27:12 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="CFAlogin()[pri:2, instance:testScript.CallLogLookerReport@3701eaf6]" name="CFAlogin" duration-ms="5198" started-at="2021-03-08T08:27:12 IST" finished-at="2021-03-08T08:27:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CFAlogin -->
        <test-method status="PASS" signature="NavigateToReportsPage()[pri:3, instance:testScript.CallLogLookerReport@3701eaf6]" name="NavigateToReportsPage" duration-ms="6189" started-at="2021-03-08T08:27:17 IST" finished-at="2021-03-08T08:27:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NavigateToReportsPage -->
        <test-method status="PASS" signature="SelectCallLogsReportFromSelectAReportList()[pri:4, instance:testScript.CallLogLookerReport@3701eaf6]" name="SelectCallLogsReportFromSelectAReportList" duration-ms="5048" started-at="2021-03-08T08:27:23 IST" finished-at="2021-03-08T08:27:28 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SelectCallLogsReportFromSelectAReportList -->
        <test-method status="FAIL" signature="SelectIsintheLastOptionFromDateRange()[pri:5, instance:testScript.CallLogLookerReport@3701eaf6]" name="SelectIsintheLastOptionFromDateRange" duration-ms="30385" started-at="2021-03-08T08:27:28 IST" finished-at="2021-03-08T08:27:59 IST">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//h1[text()='Call Logs - Enhanced']"}
  (Session info: chrome=89.0.4389.82)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'LAPTOP-OF0U061G', ip: '192.168.43.239', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_281'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 89.0.4389.82, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\samiksha\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:58788}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 0a06b363df7881201d079768eba0f1e6
*** Element info: {Using=xpath, value=//h1[text()='Call Logs - Enhanced']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//h1[text()='Call Logs - Enhanced']"}
  (Session info: chrome=89.0.4389.82)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'LAPTOP-OF0U061G', ip: '192.168.43.239', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_281'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 89.0.4389.82, chrome: {chromedriverVersion: 89.0.4389.23 (61b08ee2c5002..., userDataDir: C:\Users\samiksha\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:58788}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 0a06b363df7881201d079768eba0f1e6
*** Element info: {Using=xpath, value=//h1[text()='Call Logs - Enhanced']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at testScript.CallLogLookerReport.SelectIsintheLastOptionFromDateRange(CallLogLookerReport.java:82)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SelectIsintheLastOptionFromDateRange -->
        <test-method status="SKIP" signature="SelectNoOfDaysFromDateRange()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6]" name="SelectNoOfDaysFromDateRange" duration-ms="0" started-at="2021-03-08T08:27:59 IST" depends-on-methods="testScript.CallLogLookerReport.SelectIsintheLastOptionFromDateRange" finished-at="2021-03-08T08:27:59 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CallLogLookerReport.SelectNoOfDaysFromDateRange()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CallLogLookerReport.SelectNoOfDaysFromDateRange()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SelectNoOfDaysFromDateRange -->
        <test-method status="SKIP" signature="selectPeriod()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6]" name="selectPeriod" duration-ms="0" started-at="2021-03-08T08:27:59 IST" depends-on-methods="testScript.CallLogLookerReport.SelectNoOfDaysFromDateRange" finished-at="2021-03-08T08:27:59 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CallLogLookerReport.selectPeriod()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CallLogLookerReport.selectPeriod()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- selectPeriod -->
        <test-method status="SKIP" signature="fetchResults()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6]" name="fetchResults" duration-ms="0" started-at="2021-03-08T08:27:59 IST" depends-on-methods="testScript.CallLogLookerReport.selectPeriod" finished-at="2021-03-08T08:27:59 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method CallLogLookerReport.fetchResults()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method CallLogLookerReport.fetchResults()[pri:0, instance:testScript.CallLogLookerReport@3701eaf6] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fetchResults -->
      </class> <!-- testScript.CallLogLookerReport -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
